rke/k8s_docker_info.go

```diff
@@ -17,8 +17,8 @@ func loadK8sVersionDockerInfo() map[string][]string {
 		"1.20": {"1.13.x", "17.03.x", "17.06.x", "17.09.x", "18.06.x", "18.09.x", "19.03.x", "20.10.x"},
 		"1.21": {"1.13.x", "17.03.x", "17.06.x", "17.09.x", "18.06.x", "18.09.x", "19.03.x", "20.10.x"},
 		"1.22": {"1.13.x", "17.03.x", "17.06.x", "17.09.x", "18.06.x", "18.09.x", "19.03.x", "20.10.x"},
-		"1.23": {"1.13.x", "17.03.x", "17.06.x", "17.09.x", "18.06.x", "18.09.x", "19.03.x", "20.10.x"},
-		"1.24": {"1.13.x", "17.03.x", "17.06.x", "17.09.x", "18.06.x", "18.09.x", "19.03.x", "20.10.x"},
-		"1.25": {"1.13.x", "17.03.x", "17.06.x", "17.09.x", "18.06.x", "18.09.x", "19.03.x", "20.10.x"},
+		"1.23": {"1.13.x", "17.03.x", "17.06.x", "17.09.x", "18.06.x", "18.09.x", "19.03.x", "20.10.x", "23.0.x"},
```

Comment:
@doflamingo721 , could you please clarify what's the reasoning for adding it starting with 1.23? Not saying that it's wrong, just wondering about the reasons.

Reply:
@snasovich I added the support starting with 1.23 because rke 1.4 only supports k8s version 1.23 and above. We don't have an option to provision an older k8s version than 1.23 as per my understanding. I can update my PR if this needs to be added to the older k8s versions as well.

Reply:
@doflamingo721 , ah, make sense. Thank you for explaining.